uk.co.compucorp.civicrm.tasksassignments/js/src/tasks-assignments/controllers/modal/modal-document.js

```diff
@@ -208,14 +208,32 @@ define([
 
                 return result;
               }).then(function (result) {
+                if(!!result.files[0].result){
+                  $scope.$broadcast('ct-spinner-show');
+                  DocumentService.save({
+                      id: result.document.id,
+                      status_id: '1'
```

Comment:
ps comment what 1 is or use constant naming like `var AWAITING_APPROVAL = '1'`, same for others below